src/components/PhotoGallery/PhotoGallery.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import axios from "axios";
+import Gallery from "react-photo-gallery";
+import { shuffleArray } from "utility/Helpers";
+
+const PhotoGallery = () => {
+  const [images, setImages] = React.useState<any[]>([]);
+
+  async function fetchImages() {
+    let response = await axios.get(
+      "https://google-photos-album-demo2.glitch.me/SkVei5N56poqTh8g8"
```

Comment:
This is an external endpoint I am using. It could theoretically stop working correctly at any time. We have 3 options from the top of my head to mitigate/negate this:

- Make the default case return null if we fail to fetch any images using this API (so no images being displayed) - this is the current behavior
- Make the default case return a static list of images stored locally in our codebase - good fallback because we'll at least be displaying something if the API fails but this list will definitely be outdated as it will not be regularily maintained.
- Make our own endpoint to fetch images from our Google Photos library - something like an exact copy of what is being done here: https://medium.com/@ValentinHervieu/how-i-used-google-photos-to-host-my-website-pictures-gallery-d49f037c8e3c